.eslintrc.js

```diff
@@ -1,13 +1,9 @@
 module.exports = {
-  extends: ["react-app", "prettier"],
+  extends: ["react-app"],
   ignorePatterns: ["/build"],
   env: {
     node: true,
   },
-  plugins: ["prettier"],
-  rules: {
-    "prettier/prettier": "error",
```

Comment:
What do you think about making this a warning instead of error?

Reply:
I think it is valuable to have integrated options to fix formatting as you are editing code. You could install a separate VSCode prettier function, but fixing at commit time without user involvement is too late.

Reply:
I see your point :) I'll close this PR.